Backend/app.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const recipe = require('./model/model');
```

Comment:
nit: We usually use PascalCase for models and camelCase for model instances
```suggestion
const Recipe = require('./model/model');
```